src/components/settings/character.tsx

```
@@ -491,6 +520,50 @@ const Character = () => {
           </TextButton>
         </div>
+        <div className="my-16 whitespace-pre-line">
+          {t('characterpresetInfo')}
+        </div>
+        <div className="my-24 mb-8 flex flex-wrap gap-4">
+          {characterPresets.map(({ key, value }, index) => (
+            <div key={key} className="relative">
+              <TextButton
+                onClick={(e) => {
+                  if (e.shiftKey) {
+                    settingsStore.setState({ [key]: systemPrompt });
```

ESLint

[error] Delete `;`

(prettier/prettier)